src/onboarding/onboarding-core-config.ts

```diff
@@ -401,7 +426,7 @@ class OnboardingCoreConfig extends LitElement {
       }
       this._language = getLocalLanguage();
     } catch (err: any) {
-      alert(`Failed to detect location information: ${err.message}`);
+      this._error = `Failed to detect location information: ${err.message}`;
```

Comment:
Is the untranslated part necessary or just:

```suggestion
      this._error = err.message;
```

Reply:
🤷🏻‍♂️ the error will be in English too